File: 240411/T를 초과하는 연속 부분 수열/subsequence-above-t.py
N, T = map(int, input().split())
n_lst = list(map(int, input().split()))

answer, cnts, num = 0, 0, 0
for i in range(N):
    n_new = n_lst[i]
    if n_new > T:
        cnts += 1
    else:
        cnts = 0

    answer = max(answer, cnts)
    num = n_new

print(answer)

File: 260124/T를 초과하는 연속 부분 수열/subsequence-above-t.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int N;
    long long T;
    cin >> N >> T;

    vector<long long> n_lst(N);
    for (int i = 0; i < N; i++) cin >> n_lst[i];

    int answer = 0, cnts = 0;
    for (int i = 0; i < N; i++) 
    {
        long long n_new = n_lst[i];
        if (n_new > T) cnts++;
        else cnts = 0;

        answer = max(answer, cnts);
    }

    cout << answer << "\n";
    return 0;
}